beat/heart.py

```diff
@@ -3281,11 +3355,24 @@ def gamma(takeoffangles, azimuths):
     sa = num.sin(azimuths)
     return num.array([st*ca, st*sa, ct])
 
+def symmat6(a_xx, a_yy, a_zz, a_xy, a_xz, a_yz):
```

Comment:
so that function exists already in pyrocko why did you copy it here?

Reply:
> so that function exists already in pyrocko why did you copy it here? needs to be removed again
done!